demoextendsymfonyform3/demoextendsymfonyform3.php

```diff
@@ -0,0 +1,344 @@
+<?php
+/**
+ * 2007-2020 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License 3.0 (AFL-3.0).
+ * It is also available through the world-wide-web at this URL: https://opensource.org/licenses/AFL-3.0
+ */
+
+use DemoCQRSHooksUsage\Domain\Reviewer\Command\UpdateIsAllowedToReviewCommand;
+use DemoCQRSHooksUsage\Domain\Reviewer\Exception\CannotCreateReviewerException;
+use DemoCQRSHooksUsage\Domain\Reviewer\Exception\CannotToggleAllowedToReviewStatusException;
+use DemoCQRSHooksUsage\Domain\Reviewer\Exception\ReviewerException;
+use DemoCQRSHooksUsage\Domain\Reviewer\Query\GetReviewerSettingsForForm;
+use DemoCQRSHooksUsage\Domain\Reviewer\QueryResult\ReviewerSettingsForForm;
+use Doctrine\DBAL\Query\QueryBuilder;
+use PrestaShop\PrestaShop\Core\CommandBus\CommandBusInterface;
+use PrestaShop\PrestaShop\Core\Domain\Customer\Exception\CustomerException;
+use PrestaShop\PrestaShop\Core\Grid\Column\Type\Common\ToggleColumn;
+use PrestaShop\PrestaShop\Core\Grid\Definition\GridDefinitionInterface;
+use PrestaShop\PrestaShop\Core\Grid\Filter\Filter;
+use PrestaShop\PrestaShop\Core\Search\Filters\CustomerFilters;
+use PrestaShopBundle\Form\Admin\Type\SwitchType;
+use PrestaShopBundle\Form\Admin\Type\YesAndNoChoiceType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Session\Flash\FlashBagInterface;
+
+/**
+ * Class DemoExtendSymfonyForm1 demonstrates the usage of CQRS pattern and hooks.
+ */
+class DemoExtendSymfonyForm3 extends Module
```

Comment:
Same comment as here https://github.com/PrestaShop/example-modules/pull/35#discussion_r561705943

I think it's a good thing to split into two modules to make it less confusing, but we really need to improve the naming of these two module Or it will remain confusing and they won't meet the expected objective

Suggested names for this one:

- `DemoExtendSymfonyPageWithCQRS`
- `DemoExtendWithCQRS`
- `DemoCQRSPattern`
- `DemoCQRSForExtension`

Reply:
@jolelievre I'll perform the renaming with the improvements suggested by @kpodemski in a 2nd PR to keep it scoped 😉  and I created an issue for global renaming https://github.com/PrestaShop/example-modules/issues/40